org.eevolution.manufacturing/src/main/java/base/org/eevolution/model/MPPForecastDefinitionLine.java

```diff
@@ -108,10 +112,12 @@ public String getSQlWhere(int M_Warehouse_ID, String alias) {
 			whereClause.append("=").append(getC_BPartner_ID()).append(" AND ");
 		}
 		if (getC_BP_Group_ID() > 0) {
-			whereClause.append(alias).append(".");
-			whereClause.append(MSalesHistory.COLUMNNAME_C_BP_Group_ID);
-			whereClause.append("=").append(getM_Product_Group_ID())
-					.append(" AND ");
+			whereClause.append(alias)
+				.append(".")
+				.append(MSalesHistory.COLUMNNAME_C_BP_Group_ID)
```

Comment:
Please let all as now and just change **getM_Product_Group_ID()** by **getC_BP_Group_ID()** is more easy debug and clear

Reply:
The changes you requested were applied to make the difference between files cleaner.